### PR TITLE
Ccg latest

### PR DESCRIPTION
**Pull request by danyaljj**
Same as #363 

Incremented the illinois-cogcomp-nlp version to the latest and had to fix some of the tests. 

FYI @bhargav

### CODE REVIEW

File: saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/ace/ACEReader.scala [hunk 1]
```diff
@@ -1,123 +0,0 @@
...
-
-  val baseDir = "/Users/sameer/Work/data/ace/ace-2005/data/"
-  val dtdFile = "/Users/sameer/Work/data/ace/ace-2005/dtd/apf.v5.1.1.dtd"
-  val docDirInput = baseDir + "English/"
-  val docDirOuput = baseDir + "cache/"
-  new File(docDirOuput).mkdirs
-  val taBuilder = new CcgTextAnnotationBuilder(new IllinoisTokenizer())
-  val functor = new AceFileProcessor(taBuilder)
-  annotateAllDocument(functor, docDirInput, docDirOuput, new File(dtdFile))
-}
```
Comment (kordjamshidi) [inline saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/ace/ACEReader.scala]: Didn't we need these files for the ACE RE models?

Comment (danyaljj) [inline saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/ace/ACEReader.scala]: They are never used in any applications (not even in [RE application](https://github.com/IllinoisCogComp/saul/pull/350/files))

Comment (bhargav) [inline saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/ace/ACEReader.scala]: We have a new ACEReader in illinois-cogcomp-nlp module that is used for ACE RE.

**Comment by danyaljj** [comment]
@kordjamshidi after this PR all the ccg dependencies will be the latest.

**Comment by bhargav** [comment]
Looks good to me. 👍

**Pull request merged**